client/my-sites/plugins/notices.jsx

```diff
@@ -3,160 +3,153 @@
-		this.state = { notices: this.refreshPluginNotices() };
+	getPluginById( pluginId ) {
+		return this.props.plugins.find( ( plugin ) => plugin.id === pluginId );
 	}
 
-	componentDidMount() {
-		PluginsLog.on( 'change', this.showNotification );
+	getCombination( translateArg ) {
+		return (
+			( translateArg.numberOfSites > 1 ? 'n sites' : '1 site' ) +
+			' ' +
+			( translateArg.numberOfPlugins > 1 ? 'n plugins' : '1 plugin' )
+		);
 	}
 
-	componentWillUnmount() {
-		PluginsLog.removeListener( 'change', this.showNotification );
-	}
+	getTranslateArg( logs, sampleLog, typeFilter ) {
+		const filteredLogs = logs.filter( ( log ) => {
+			return log.status === typeFilter ? typeFilter : sampleLog.type;
+		} );
 
-	shouldComponentUpdateNotices( currentNotices, nextNotices ) {
-		if ( currentNotices.errors && currentNotices.errors.length !== nextNotices.errors.length ) {
-			return true;
-		}
-		if (
-			currentNotices.inProgress &&
-			currentNotices.inProgress.length !== nextNotices.inProgress.length
-		) {
-			return true;
-		}
-		if (
-			currentNotices.completed &&
-			currentNotices.completed.length !== nextNotices.completed.length
-		) {
-			return true;
-		}
-		return false;
-	}
+		const numberOfSites = uniqBy( filteredLogs, 'siteId' ).length;
+		const numberOfPlugins = uniqBy( filteredLogs, 'pluginId' ).length;
+
+		const logSite = this.props.sites.find( ( site ) => parseInt( sampleLog.siteId ) === site?.ID );
+		const logPlugin = this.getPluginById( sampleLog.pluginId );
 
-	refreshPluginNotices = () => {
-		const site = this.props.selectedSite;
 		return {
-			errors: filterNotices( PluginsLog.getErrors(), site, this.props.pluginSlug ),
-			inProgress: filterNotices( PluginsLog.getInProgress(), site, this.props.pluginSlug ),
-			completed: filterNotices( PluginsLog.getCompleted(), site, this.props.pluginSlug ),
+			plugin: logPlugin?.name,
+			numberOfPlugins,
+			site: logSite?.title,
+			isMultiSite: logSite?.is_multi_site,
+			numberOfSites,
 		};
-	};
+	}
 
 	showNotification = () => {
-		const logNotices = this.refreshPluginNotices();
-		this.setState( { notices: logNotices } );
-		if ( logNotices.inProgress.length > 0 ) {
-			notices.info(
-				this.getMessage( logNotices.inProgress, this.inProgressMessage, 'inProgress' )
+		const currentNotices = this.getCurrentNotices();
+
+		if ( currentNotices.inProgress.length > 0 ) {
+			this.props.infoNotice(
+				this.getMessage( currentNotices.inProgress, this.inProgressMessage, 'inProgress' ),
+				{
+					id: 'plugin-notice',
+				}
 			);
 			return;
 		}
 
-		if ( logNotices.completed.length > 0 && logNotices.errors.length > 0 ) {
-			notices.warning( this.erroredAndCompletedMessage( logNotices ), {
-				onRemoveCallback: () => reduxDispatch( removePluginStatuses( 'completed', 'error' ) ),
-			} );
-		} else if ( logNotices.errors.length > 0 ) {
-			notices.error( this.getMessage( logNotices.errors, this.errorMessage, 'error' ), {
-				button: this.getErrorButton( logNotices.errors ),
```

Comment:
Seems like none of the errors that we're keeping have a button or href (we're removing the obsolete notice in this PR, I've explained why below), so we can totally remove those.